cor_graphical/verification/verification_report.Rmd

```diff
@@ -0,0 +1,66 @@
+---
+title: 'Verification Report: correlates_graphical'
+author: "Di Lu"
+date: "2021/03/27"
+output:
+  html_document: default
+  pdf_document: default
+---
+
+## Description
+
+This is the verification report for the `cor_graphical` folder of the `correlates_reporting` project for CoVPN. 
+
+In the verification process, the output of `cor_graphical_verification_code.R` is compared against the plots listed in the the folder `immuno_graphical/verification/verification_input`. The values presented in the original plot against the ones generated by the verification code. Any statistical calculations will either be verified by computing them to a verified statistical table , or they will be generated independently.
+
+The files `cor_graphical/verification/verification_input/practice_data.csv`, `cor_graphical/verification/verification_input/longer_cor_data_plot1.rds` and 
+`cor_graphical/verification/verification_input/plot.25sample1.rds` was provided by the original programmer to the tester for verification purposes.
+
+####lineplots of Binding Antibody to Spike: baseline negative vaccine arm (2 timepoints)
+Outputs of Line plots, Violin plots and box plots and were visually compared to the outputs generated by primary programmer, one issue that was identified was the titer values were turncated by wrong ULOQ, so the verification is pending.
```

Comment:
Please add a programmatic comparison of the input data & calculated statistics you generated against the data 
 & stats provided by kendrick. (save the input data for each plot you generated against the input data kendrick generated for each plot and compare via `identical()` or `dplyr::all_equal()`.